sopel/db.py

```diff
@@ -332,11 +406,6 @@ def get_nick_id(self, nick: str, create: bool = False) -> int:
                 session.add(nickname)
                 session.commit()
             return nickname.nick_id
-        except SQLAlchemyError:
-            session.rollback()
-            raise
```

Comment:
I understand removing the `finally` block below (its function is now served by the context manager), but what am I missing that replaces this error handler?

This applies to other similar functions below, but I only need the question answered once. 😁

Reply:
The [context manager does both](https://docs.sqlalchemy.org/en/14/orm/session_transaction.html#managing-transactions) now!